tasks/setup-debian.yml

```diff
@@ -1,32 +1,15 @@
 ---
-- name: Setup WireGuard preference
-  copy:
-    src: debian/etc/apt/preferences.d/limit-unstable
-    dest: /etc/apt/preferences.d/limit-unstable
```

Comment:
I guess we can assume that this file already exists if a user used that role already. So it should be mentioned at least in the changelog to delete this file manually or use a task to delete it.

Reply:
It is a scary one. If you remove the pin but not the repo, you will start updating from unstable (a bad idea).

If you remove the repo + the pin that will work. I'll add this to the changelog.